.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: Dart
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image:  google/dart:beta
```

Comment:
Maybe this can now be bumped to `dev`?

Reply:
Do we have a new dev release out?

Reply:
It doesn't look like we're ready for this yet.